diagnostic.md

```diff
@@ -45,6 +50,8 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
+true
+false
 
```

Comment:
`weather = "sunny";` - assigns the value `"sunny"` to the variable `weather`
` weather === "sunny";` - a Boolean expression that checks if `weather` is exactly equal to `"sunny"`, which would evaluate to `true`